### DISCUSSION TITLE
LND Setup

### DISCUSSION THREAD

**Discussion by Scully78** [raspiblitz/raspiblitz#4797] [General]
I'm trying to set up LND but i keep getting a message when i try to load or create a wallet.  
[lncli] rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing: dial tcp 0.0.0.0 :10009: connect: connection refused".

I've tried altering the lnd.conf file with various suggestions I've found elsewhere but nothing seems to work.

**Comment by rootzoll** [comment]
Best would be to post a debug log here so we can see the state of your system:
https://docs.raspiblitz.org/docs/faq#debug
